standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/CompactionTxnHandler.java

```diff
@@ -993,7 +994,7 @@ public boolean checkFailedCompactions(CompactionInfo ci) throws MetaException {
     try {
       try {
         dbConn = getDbConn(Connection.TRANSACTION_READ_COMMITTED);
-        pStmt = dbConn.prepareStatement("SELECT \"CC_STATE\" FROM \"COMPLETED_COMPACTIONS\" WHERE " +
+        pStmt = dbConn.prepareStatement("SELECT \"CC_STATE\", \"CC_ENQUEUE_TIME\" FROM \"COMPLETED_COMPACTIONS\" WHERE " +
```

Comment:
Is there a reason for using CC_ENQUEUE_TIME over CC_END (or CC_START)?

Reply:
enqueue time is guaranteed to be filled out for every compaction, the others might be null for a failed one.